go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/hashicorp/go-retryablehttp v0.7.4
 	github.com/hashicorp/go-version v1.6.0
 	github.com/hashicorp/nomad/api v0.0.0-20220506174431-b5665129cd1f
+	github.com/http-wasm/http-wasm-host-go v0.5.2-0.20231031144646-6e96a183660d
```

Comment:
@zetaab do you if it is planned to have a release v0.5.2  version

Reply:
It is out now https://github.com/http-wasm/http-wasm-host-go/releases/tag/v0.5.2

Reply:
cc @zetaab @ldez